swan-lake/featured-use-cases/write-a-restful-api-with-ballerina.md

```diff
@@ -155,31 +156,14 @@ resource function post countries(@http:Payload CovidEntry[] covidEntries)
         };
     } else {
         covidEntries.forEach(covdiEntry => covidTable.add(covdiEntry));
-        return <CreatedCovidEntries>{body: covidEntries};
+        return covidEntries;
     }
 }
 ```
 
 In this code:
 
-- It is chosen to either accept the entire payload or send back an error. Copying this straightway results in an error, which is expected as the `CreatedCovidEntries` and `ConflictingIsoCodesError` types are not defined yet.
-
-#### Define the record type
-
-To define the `CreatedCovidEntries` record, add the code below to the API template file (i.e., `service.bal`).
-
-```ballerina
-public type CreatedCovidEntries record {|
-   *http:Created;
-   CovidEntry[] body;
-|};
-```
-
-In this code:
-- This resource has a resource argument named `covidEntries` annotated with `@http:Payload`. This means the resource is expecting a payload with type `CovideEntry[]`. There are two types of records `CreatedCovidEntries` and `ConflictingIsoCodesError` as the return values. Following is the definition of `CreatedCovidEntries`.
-- `*http:Created` is the Ballerina way of saying one type is a subtype of another. In this case, `CreatedCovidEntries` is a subtype of `*http:Created`.
-- Ballerina has defined a set of types for each HTTP status code. This allows you to write services in a type-oriented way, which in turn is helpful when it comes to tooling and generating OpenAPI specifications for HTTP services. 
```

Comment:
We need following 3 lines also

Reply:
Fixed with [2239321437995ac6db1cf3a3d85c4c61bad262e8](https://github.com/ballerina-platform/ballerina-dev-website/pull/5690/commits/2239321437995ac6db1cf3a3d85c4c61bad262e8)